react/features/base/config/interfaceConfigWhitelist.js

```diff
@@ -31,7 +31,6 @@ export default [
     'GENERATE_ROOMNAMES_ON_WELCOME_PAGE',
     'HIDE_INVITE_MORE_HEADER',
     'INDICATOR_FONT_SIZES',
-    'INITIAL_TOOLBAR_TIMEOUT',
```

Comment:
We should keep them whitelisted for backwards compat.

Reply:
this overwrite never worked, as the toolbox store was initializing before the overwritten configs would be set.